src/usr/local/www/interfaces.php

```diff
@@ -1702,7 +1702,7 @@ function build_gatewayv6_list() {
 
 if (is_subsystem_dirty('interfaces')) {
 	print_apply_box(sprintf(gettext("The %s configuration has been changed."), $wancfg['descr']) . "<br />" .
-					gettext("You must apply the changes in order for them to take effect.") . "<br />" .
+					gettext("The changes must be applied in order for them to take effect.") . "<br />" .
```

Comment:
Still seems wordy... "The changes must be applied to take effect" gets the point across, or even "The changes must be applied"